### PR TITLE
Pentium basic

### PR DESCRIPTION
**Pull request by cacad101**
Basic function
- Login
- Signup
- Logout
- View Home

### CODE REVIEW

File: storage/db.go [hunk 1]
```diff
@@ -1,44 +1,45 @@
 package storage
 
 import (
-    "github.com/jmoiron/sqlx"
-    "github.com/satori/go.uuid"
-    _ "github.com/mattn/go-sqlite3"
-    "../models"
+	"github.com/cacad-ntu/GameMaster/models"
```
Comment (dbakti7) [inline storage/db.go:4]: 2 tabs? @cacad101

File: storage/user.go [hunk 2]
```diff
@@ -2,7 +2,8 @@ package storage
 
 import (
 	"database/sql"
-	"../models"
+
+	"github.com/cacad-ntu/GameMaster/models"
```
Comment (dbakti7) [inline storage/user.go:6]: Thanks for fixing this, too lazy at that time. lol

File: resources/styles.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto);
+
+/****** LOGIN MODAL ******/
+.login-container {
+  padding: 30px;
```
Comment (dbakti7) [inline resources/styles.css]: Fix indent.

File: handler/user.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+import (
+	"net/http"
+
+	"github.com/cacad-ntu/GameMaster/controllers"
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+const (
+	secretKey = "diangoblok"
```
Comment (dbakti7) [inline handler/user.go]: change this, else your PR will be rejected. @cacad101

**Review by dbakti7 — CHANGES_REQUESTED** [review]

File: storage/db.go (hunk 1 above)
Comment (cacad101) [reply storage/db.go:4]: Not sure about this on local are using 4 space indent

File: resources/styles.css (hunk 3 above)
Comment (cacad101) [reply resources/styles.css]: Fixed

File: handler/user.go (hunk 4 above)
Comment (cacad101) [reply handler/user.go]: Changed

File: handler/user.go [hunk 5]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	secretKey = "diangoblok"
+	secretKey = "dian-goblok"
 	issuer    = "cacad-ntu"
```
Comment (dbakti7) [inline handler/user.go]: Not-approved! @cacad101 
I suggest "pentium-inside"

File: storage/db.go (hunk 1 above)
Comment (dbakti7) [reply storage/db.go:4]: I will try fixing it and standardize our setting after this PR merged.

**Review by dbakti7 — COMMENTED** [review]
Logic wise seems fine. Change the secretkey and merge it! @cacad101

File: html/home.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
```
Comment (dbakti7) [inline html/home.html:5]: Small stuff, can be TODO: add title later. @cacad101

File: html/login.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
```
Comment (dbakti7) [inline html/login.html:5]: Same here, title. @cacad101

File: html/home.html (hunk 6 above)
Comment (cacad101) [reply html/home.html:5]: Okay

File: html/login.html (hunk 7 above)
Comment (cacad101) [reply html/login.html:5]: Okay

File: storage/db.go (hunk 1 above)
Comment (cacad101) [reply storage/db.go:4]: Okay

**Pull request merged**